src/tools/info/tl_info.c

```diff
@@ -143,8 +143,17 @@ static void print_iface_info(uct_worker_h worker, uct_md_h md,
         return;
     }
 
-    printf("#   Transport: %s\n", resource->tl_name);
-    printf("#      Device: %s\n", resource->dev_name);
+    printf("#      Transport: %s\n", resource->tl_name);
+    printf("#         Device: %s\n", resource->dev_name);
+
+    printf("#  System device: ");
```

Comment:
can be simplified:
```
printf("#  System device: %s (%d)\n",
       ucs_topo_sys_device_bdf_name(resource->sys_device, buf, sizeof(buf)),
       resource->sys_device);
```

since `ucs_topo_sys_device_bdf_name()` handles the case when a passed `sys_device` is `UCS_SYS_DEVICE_ID_UNKNOWN`

Reply:
in case of UNKNOWN, i wanted to avoid printing the numeric sys_dev with %d

Reply:
i see
then better to do:
```
printf("#  System device: %s",
       ucs_topo_sys_device_bdf_name(resource->sys_device, buf, sizeof(buf)));
if (resource->sys_device != UCS_SYS_DEVICE_ID_UNKNOWN) {
    printf(" (%d)", resource->sys_device);
}
printf("\n");
```

wdyt?

Reply:
fixed